### ISSUE TITLE
Simplify Google STT request code

### ISSUE THREAD

**Issue by janakj** [JQSELMAQ/experiment_run_robot#8]
https://github.com/JQSELMAQ/experiment_run_robot/blob/27643c50613bf5ef42308dd5beb8a81c48ae0e3b/robot.py#L192-L217

This code can be written in a more succinct way. Note that you have an if..else statement here and the blocks in the if and else parts are almost identical. They differ only in the format of the JSON file submitted to Google. In one branch, the JSON document has a "speechContexts" property, in the other it does not. You can remove a lot of the repetition by rewriting the code as follows:

```Python
data = {
    "config": { 
        "encoding": "LINEAR16", 
        "languageCode": "en-US" 
    }, 
    "audio": { 
        "content": lists[cur_item]}
    }
}

if nato_tune is True:
    data["config"]["speechContexts"] = [{ 
        "phrases": [nato_list], 
        "boost": 100 
    }]

request = requests.post(API_URL, json=data) 
# encoded files of items 
```
In other words, build the JSON document in a separate variable that is passed to requests.posts. If nato_tune is set, add the "speechContexts" property to the JSON document. No need for the top-level if-else statement anymore and all duplicate code is gone.
